tests/tck/features/expressions/typeConversion/TypeConversion1.feature

```diff
@@ -101,5 +96,5 @@ Feature: TypeConversion1 - To Boolean
       | invalid |
       | []      |
       | {}      |
-      | 1       |
+      # | 1       |
```

Comment:
take a look at https://github.com/opencypher/openCypher/blob/master/tck/features/expressions/typeConversion/TypeConversion1.feature look like this file was updated